yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1233,22 +1233,22 @@ func generateAnalyzeSchemaReport(msr *metadb.MigrationStatusRecord, reportFormat
 }
 
 // analyze issue reasons to modify the reason before sending to callhome as will have sensitive information
-var reasonsIncludingSensitiveInformationToCallhome = []string{
-	UNSUPPORTED_PG_SYNTAX_ISSUE_REASON,
-	queryissue.STORED_GENERATED_COLUMNS_ISSUE_DESCRIPTION,
-	queryissue.POLICY_ROLE_ISSUE_DESCRIPTION,
-	queryissue.INSUFFICIENT_COLUMNS_IN_PK_FOR_PARTITION_ISSUE_DESCRIPTION,
-	queryissue.XML_DATATYPE_ISSUE_DESCRIPTION,
-	queryissue.XID_DATATYPE_ISSUE_DESCRIPTION,
-	queryissue.POSTGIS_DATATYPE_ISSUE_DESCRIPTION,
-	queryissue.UNSUPPORTED_DATATYPE_ISSUE_DESCRIPTION,
-	queryissue.UNSUPPORTED_DATATYPE_LIVE_MIGRATION_ISSUE_DESCRIPTION,
-	queryissue.UNSUPPORTED_DATATYPE_LIVE_MIGRATION_WITH_FF_FB_ISSUE_DESCRIPTION,
-	queryissue.PK_UK_ON_COMPLEX_DATATYPE_ISSUE_DESCRIPTION,
-	queryissue.INDEX_ON_COMPLEX_DATATYPE_ISSUE_DESCRIPTION,
-	queryissue.LARGE_OBJECT_DATATYPE_ISSUE_DESCRIPTION,
-	queryissue.MULTI_RANGE_DATATYPE_ISSUE_DESCRIPTION,
-}
+// var reasonsIncludingSensitiveInformationToCallhome = []string{
+// 	UNSUPPORTED_PG_SYNTAX_ISSUE_REASON,
+// 	queryissue.STORED_GENERATED_COLUMNS_ISSUE_DESCRIPTION,
+// 	queryissue.POLICY_ROLE_ISSUE_DESCRIPTION,
+// 	queryissue.INSUFFICIENT_COLUMNS_IN_PK_FOR_PARTITION_ISSUE_DESCRIPTION,
+// 	queryissue.XML_DATATYPE_ISSUE_DESCRIPTION,
+// 	queryissue.XID_DATATYPE_ISSUE_DESCRIPTION,
+// 	queryissue.POSTGIS_DATATYPE_ISSUE_DESCRIPTION,
+// 	queryissue.UNSUPPORTED_DATATYPE_ISSUE_DESCRIPTION,
+// 	queryissue.UNSUPPORTED_DATATYPE_LIVE_MIGRATION_ISSUE_DESCRIPTION,
+// 	queryissue.UNSUPPORTED_DATATYPE_LIVE_MIGRATION_WITH_FF_FB_ISSUE_DESCRIPTION,
+// 	queryissue.PK_UK_ON_COMPLEX_DATATYPE_ISSUE_DESCRIPTION,
+// 	queryissue.INDEX_ON_COMPLEX_DATATYPE_ISSUE_DESCRIPTION,
+// 	queryissue.LARGE_OBJECT_DATATYPE_ISSUE_DESCRIPTION,
+// 	queryissue.MULTI_RANGE_DATATYPE_ISSUE_DESCRIPTION,
+// }
 
 // analyze issue reasons to send the object names for to callhome
 var reasonsToSendObjectNameToCallhome = []string{
```

Comment:
We can use the issue.Type here instead of Reason to avoid using the Reason now and also where ever we now use this slice we could easily do `slices.Contains(reasonsToSendObjectNameToCallhome, issue.Type)` ?